### ISSUE TITLE
Should the algorithm avoid corner cases

### ISSUE THREAD

**Issue by Brli** [lucasnlm/antimine-android#417]
![Screenshot_20230307-231045_ASUS_Launcher.png](https://user-images.githubusercontent.com/4042566/223463663-54829b39-3ff1-4ec3-9df0-531203d696c0.png)

Like this, the corner is surrounded with mines

**Comment by SuspectTyrannosaurus** [comment]
This is also the issue raised in #394

**Comment by lucasnlm** [comment]
I'm not sure if this is a problem. There are zero remaining mines, so you can open the corners without problem.

**Comment by Brli** [comment]
@lucasnlm: That is where I've managed to solved rest of the part, to just show you that there are boxes left surrounded with mines.

Check out the case where @SuspectTyrannosaurus pointed

The problem is, without the left mine indicator, there is no hint for the boxes, which becomes pure luck.

**Comment by lucasnlm** [comment]
@Brli I agree partially. But this is something that other minesweeper implementation does. So, I'm not sure if remove it is something good.

**Comment by PeterBindels-TomTom** [comment]
IMO this is fine. I had a case where the mines-left indicator was needed for one corner to know you had to click the corner one. Solvable is solvable.
